views/partials/subpages/confirm.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="dashboard-email-sent bold"> {{ getString "email-verified" }}</span>
+<h1>{{ getString "email-added-to-subscription" email=email }}</h1>
+{{{ getString "email-verified-view-dashboard" sign-in-link='<a href="/oauth/init">Sign in</a>'}}}
```

Comment:
@lesleyjanenorton has an idea how we can move this into its own helper so we can re-use the `sign-in` string already in the `.ftl` file.